### PR TITLE
Update Set-MpPreference.md

### PR DESCRIPTION
**Pull request by msbemba**
AddedDisable cache maintenance to the parameters

Fixes https://github.com/MicrosoftDocs/windows-powershell-docs/issues/3502

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: docset/winserver2016-ps/defender/Set-MpPreference.md [hunk 1]
```diff
@@ -236,6 +236,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisableCacheMaintenance
+Defines whether the cache maintenance idle task will perform the cache maintenance or not. Allowed values are 1- Cache maintenance is disabled and 0- Cache maintenance is enabled (default).
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/defender/Set-MpPreference.md:240]:
```suggestion
Defines whether the cache maintenance idle task will perform the cache maintenance or not. Allowed values are 1 - cache maintenance is disabled, and 0 - cache maintenance is enabled (default).
```

File: docset/winserver2019-ps/defender/Set-MpPreference.md [hunk 2]
```diff
@@ -272,6 +272,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+
+### -DisableCacheMaintenance
+Defines whether the cache maintenance idle task will perform the cache maintenance or not. Allowed values are 1- Cache maintenance is disabled and 0- Cache maintenance is enabled (default).
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/defender/Set-MpPreference.md:277]:
```suggestion
Defines whether the cache maintenance idle task will perform the cache maintenance or not. Allowed values are 1 - cache maintenance is disabled, and 0 - cache maintenance is enabled (default).
```

File: docset/winserver2022-ps/defender/Set-MpPreference.md [hunk 3]
```diff
@@ -395,6 +395,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisableCacheMaintenance
+Defines whether the cache maintenance idle task will perform the cache maintenance or not. Allowed values are 1- Cache maintenance is disabled and 0- Cache maintenance is enabled (default).
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/defender/Set-MpPreference.md:399]:
```suggestion
Defines whether the cache maintenance idle task will perform the cache maintenance or not. Allowed values are 1 - cache maintenance is disabled, and 0 - cache maintenance is enabled (default).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**